app/controllers/facility_account_price_groups_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class FacilityAccountPriceGroupsController < ApplicationController
+  admin_tab :all
+
+  before_action :authenticate_user!
+  before_action { @active_tab = "admin_users" }
+  before_action :load_resources
+  before_action :authorize_account
+
+  layout "two_column"
+
+  def show
+  end
+
+  def load_resources
```

Comment:
I think load_resources and authorize_account should be private methods as they are not controller actions